Sources/EmojiText/CADisplayLinkPublisher.swift

```diff
@@ -7,7 +7,7 @@
 
 import SwiftUI
 
-#if os(iOS) || targetEnvironment(macCatalyst) || os(tvOS)
+#if os(iOS) || targetEnvironment(macCatalyst) || os(tvOS) || os(visionOS)
```

Comment:
This is the change I've added everywhere. It'll compile with Xcode 15.0 and forward, no need to be on a beta Xcode for it to pass.